modules/ROOT/pages/clustering/databases.adoc

```diff
@@ -573,43 +575,30 @@ CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 'azb://myStorageAcco
 ======
 =====
 
-[[s3-seed-provider]]
-==== S3SeedProvider
-
-The `S3SeedProvider` supports:
-
-** `s3:` label:deprecated[Deprecated in 5.26]
+The `CloudSeedProvider` supports seeding up to a specific date or transaction ID using the `seedRestoreUntil` option.
 
+==== Seed up to a specific date
 
-[NOTE]
-====
-Neo4j 5 comes bundled with necessary libraries for AWS S3 connectivity.
-Therefore, if you use `S3SeedProvider`,`aws cli` is not required but can be used with the `CloudSeedProvider`.
-====
+To seed up to a specific date, you need to pass the differential backup, which contains the data up to that date.
 
-The `S3SeedProvider` requires additional configuration.
-This is specified with the `seedConfig` option.
-This option expects a comma-separated list of configurations.
-Each configuration value is specified as a name followed by `=` and the value, as such:
 
-[source, cypher, role="noplay"]
+[source,shell]
 ----
-CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedConfig: 'region=eu-west-1' }
+CREATE DATABASE foo OPTIONS { existingData: 'use', seedURI: 's3:/myBucket/myBackup.backup', seedRestoreUntil: datetime("2019-06-01T18:40:32.142+0100") }
 ----
 
-`S3SeedProvider` also requires passing in credentials.
-These are specified with the `seedCredentials` option.
-Seed credentials are securely passed from the Cypher command to each server hosting the database.
-For this to work, Neo4j on each server in the cluster must be configured with identical keystores.
-This is identical to the configuration required by remote aliases, see xref:database-administration/aliases/remote-database-alias-configuration.adoc#remote-alias-config-DBMS_admin-A[Configuration of DBMS with remote database alias].
-If this configuration is not performed, the `seedCredentials` option fails.
+This will seed the database with transactions committed before the provided timestamp.
```

Comment:
The 'seed up to date' functionality is not properly present in 2025.01, so this section needs to be removed. It works to seed up to a transaction id, but Cypher Operations need to make a change to allow creating a datetime with `system` as the current database to make this work nicely.

Reply:
Wait, late breaking news, they merged the change before cut-off! https://github.com/neo-technology/neo4j/pull/28684

So this section about restore until datetime should remain in the doc.

Reply:
It is also available in Cypher 5, yes.

Reply:
Confirmed, available in Cypher 5.